client/src/components/Typeahead/Typeahead.stories.tsx

```diff
@@ -43,19 +44,38 @@ const data = [
   },
 ];
 
-const filter = (query, datum) =>
-  _.includes(_.lowerCase(datum.title), _.lowerCase(query));
+const filter = (
+  query: string,
+  datum: Record<
+    string,
+    | string
+    | JSX.Element
+    | ((...args: (React.ReactNode | string | (() => void))[]) => void)
+  >
```

Comment:
Could this be
```
{
  title: string;
  [string]: string | JSX.Element | ((...args: (React.ReactNode | string | (() => void))[]) => void); // maybe reasonable for this to be type any
}
```
?

Wouldn't need to cast the `title` property to string below then.